### ISSUE TITLE
VScode extentions support

### ISSUE THREAD

**Issue by mats16** [aws-cloudformation/cloudformation-guard#12]
I think some developers want to use cloudformation-guard on [VScode](https://code.visualstudio.com/).
Is it in the roadmap?

**Comment by nathanataws** [comment]
Yes, it absolutely is.  We're exploring what an integration looks like.

**Comment by nathanataws** [comment]
I'm going to close this issue for now but rest-assured it will end up on the roadmap we're tracking in the Projects tab.

**Comment by PatMyron** [comment]
https://github.com/aws-cloudformation/cloudformation-guard/issues/49 should be done first for this
